packages/webpack/package.json

```diff
@@ -43,11 +43,13 @@
     "@vocab/types": "^1.1.2",
     "chalk": "^4.1.0",
     "debug": "^4.3.1",
-    "loader-utils": "^2.0.0",
     "virtual-resource-loader": "^1.0.1"
   },
   "devDependencies": {
     "@types/debug": "^4.1.5",
-    "@types/loader-utils": "^2.0.1"
+    "webpack": "^5.27.0"
+  },
+  "peerDependencies": {
+    "webpack": "^5.27.0"
```

Comment:
Leaving this decision as a comment so there's a record of it somewhere public.

Vocab has very few consumers outside of SEEK, and `@vocab/webpack`'s only dependent is `sku` which is already using a newer version of webpack than this. So, in the interest of not needing to do any work in `sku` to support a new major version, we've decided not to do a breaking change despite adding a peer dependency on `webpack` (that should've already been there).